### PR TITLE
YJIT: Add a hint for fixing "check-yjit-bindings"

### PR DESCRIPTION
**Pull request by XrXr**
This makes it clear what to do when the CI check
fails and should remove a few minutes of confusion for people seeing this for the first time.

---
Real estate is limited for the message:
Screenshot 2022-12-08 at 1 36 16 PM
Screenshot 2022-12-08 at 1 37 35 PM

### CODE REVIEW

**Review by maximecb — APPROVED** [review]
Thank you Alan 🙏

File: .github/workflows/yjit-ubuntu.yml [hunk 1]
```diff
@@ -47,7 +47,8 @@ jobs:
       fail-fast: false
       matrix:
         include:
-          - test_task: "check-yjit-bindings"
+          - hint: "To fix: use patch in logs" # this shows up first in the web interface
+            test_task: "check-yjit-bindings"
```
Comment (k0kubun) [inline .github/workflows/yjit-ubuntu.yml:51]:
```suggestion
          - test_task: "check-yjit-bindings"
             hint: "To fix: use patch in logs" # this shows up first in the web interface
```
Can we keep the make task name coming first? We try to make job names look like a make command `make check-yjit-bindings`, but now it looks like `make To fix: use patch in logs check-yjit-bindings`. If we want to make it a bit easier to see, maybe we could say `test_task: "yjit-bindgen"` instead.

**Comment by k0kubun** [comment]
Also, because GitHub Actions are now making automated commits in some places, I think it's also possible to push yjit-bindgen results as matzbot, at least to master, and possibly to PR branches allowing edits from maintainers as well.

**Review by k0kubun — APPROVED** [review]
:+1: Thanks

**Comment by XrXr** [comment]
Screenshot 2022-12-08 at 2 32 06 PM
Added 11c6025 and now there is enough space to at least show a part of the hint :)

**Comment by k0kubun** [comment]
:+1: Nice

**Comment by k0kubun** [comment]
> https://github.com/ruby/ruby/pull/6883/checks?check_run_id=9976692628
> /bin/sh: 1: git: not found

I guess you need to skip it when git(1) doesn't exist 🤷‍♂️

**Pull request merged**